internal/rbd/controllerserver.go

```diff
@@ -1611,36 +1613,3 @@ func (cs *ControllerServer) ControllerExpandVolume(
 		NodeExpansionRequired: nodeExpansion,
 	}, nil
 }
-
-// ControllerPublishVolume is a dummy publish implementation to mimic a successful attach operation being a NOOP.
-func (cs *ControllerServer) ControllerPublishVolume(
```

Comment:
I'm afraid this will bite you, unless you change the rbd CSIDriver object that today indicates `attachRequired: true`.

Do you happen to know if ceph rbd has a concept of an attach?
I was just looking around this area and wasn't sure why it's not doing `attachRequired: false`
and implements these.

Reply:
Good catch, thanks! I'll have a look at that too.